src/practice.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+
+const styles = {
+  container: {
+    margin: '0 auto',
+    width: '500px',
+    marginTop: '10rem'
+  },
+  menu: {
+    border: '2px solid black',
+    width: '30rem'
+  },
+  p: {
+    fontWeight: 'bold',
+    fontSize: '16px'
+  },
+  a: {
+    color: 'black',
+    cursor: 'pointer',
+    fontWeight: 'bold'
+  },
+  title: {
+    fontWeight: 'bold',
+    fontSize: '36px'
+  },
+  previous: {
+    transform: 'translateY(4.5rem) translateX(-2rem)',
+    cursor: 'pointer'
+  },
+  next: {
+    transform: 'translateY(4.5rem) translateX(29.5rem)',
```

Comment:
Instead of using a transform for horizontal positioning, try using absolute positioning. That way if you modify your container's width, you don't also have to mess with the translateX of your next/previous icons.